django/db/backends/base/schema.py

```diff
@@ -99,6 +99,8 @@ def execute(self, sql, params=()):
         Executes the given SQL statement, with optional parameters.
         """
         # Log the command we're running, then run it
+        if isinstance(sql, tuple):
+            sql, params = sql  # Used by e.g. _create_index_sql()
```

Comment:
Yes, it's just ignored for now. The only case where this happens though is with `deferred_sql` which doesn't have the concept of `(sql, params)` for whatever reason. Adding this there would be cleaner but requires more refactoring. I'll give it a shot, tho.